pom.xml

```diff
@@ -3,23 +3,27 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.2</version>
+    <version>4.16</version>
+    <relativePath/>
   </parent>
+  <artifactId>gitlab-plugin</artifactId>
+  <version>${revision}${changelist}</version>
```

Comment:
If you want to incrementalify, use

```bash
mvn incrementals:incrementalify
```

This PR is missing `.mvn/` contents. See https://github.com/jenkinsci/incrementals-tools#enabling-incrementals-the-easy-way